main/src/library/Environment.flix

```diff
@@ -123,13 +123,13 @@ mod Environment {
     /// The returned value is never less than one.
     ///
     pub def getVirtualProcessors(): Int32 =
-        unsafe Runtime.getRuntime().availableProcessors()
+        unsafely {Env, IO, Sys} run Runtime.getRuntime().availableProcessors()
```

Comment:
Its always `unsafely IO run exp` expect some places in Environment and one place ins String